internal/commands/app/diff_inputs.go

```diff
@@ -0,0 +1,19 @@
+package app
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+const (
+	flagIncludeDependencies      = "include-dependencies"
+	flagIncludeDependenciesShort = "d"
+	flagIncludeDependenciesUsage = "include to diff Realm app dependencies changes as well"
+	flagIncludeHosting           = "include-hosting"
+	flagIncludeHostingShort      = "s"
+	flagIncludeHostingUsage      = "include to diff Realm app hosting changes as well"
+)
+
+func (i *diffInputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	return nil
```

Comment:
probably want to have an `if i.AppDirectory == ""` check here, and if so, set it to `profile.WorkingDirectory`